.github/workflows/lint-js.yml

```diff
@@ -0,0 +1,37 @@
+name: Lint JS
+
+on:
+  # Run on pushes to select branches and on all pull requests.
+  push:
+    branches:
+      - master
```

Comment:
The `master` branch has been renamed to `main`.

Reply:
Good catch! Fixed :+1: